root/components/LabelsList.js

```diff
@@ -33,19 +38,71 @@ const LabelsList = ({
   labels,
   noAreas,
   noRatings,
+  order,
+  sortable,
 }: Props) => (
   <table className="tbl">
     <thead>
       <tr>
         {$c.user_exists && checkboxes ? (
           <th className="pos" />
         ) : null}
-        <th>{l('Label')}</th>
-        <th>{l('Type')}</th>
-        <th>{l('Code')}</th>
+        <th>
+          {sortable
+            ? (
+              <SortableTableHeader
+                label={l('Label')}
+                name="name"
+                order={order ? order : null}
```

Comment:
`order || null` is a bit more to-the-point though we should prob just be able to pass `order` and have SortableTableHeader handle undefined

Reply:
Added an extra commit for that